cli/tests/unit/stdio_test.ts

```diff
@@ -2,13 +2,13 @@
 import { unitTest, assertEquals } from "./test_util.ts";
 
 unitTest(async function stdioStdinRead() {
-  const nread = await Deno.stdin.read(new Uint8Array(0));
-  assertEquals(nread, 0);
+  const nread = await Deno.stdin.read(new Uint8Array(42));
+  assertEquals(nread, 42);
 });
 
 unitTest(function stdioStdinReadSync() {
-  const nread = Deno.stdin.readSync(new Uint8Array(0));
-  assertEquals(nread, 0);
+  const nread = Deno.stdin.readSync(new Uint8Array(42));
+  assertEquals(nread, 42);
```

Comment:
Hangs from a tty is true and yields EOF from CI

Reply:
Thanks